experimental/StandardFiniteFields/src/StandardFiniteFields.jl

```diff
@@ -23,10 +23,11 @@ function pop_largest_factor!(f::Fac{ZZRingElem})
 end
 
 # TODO : Should be fixed in Nemo
-function (k::Nemo.FpField)(a::Vector)
+function (k::FpField)(a::Vector)
```

Comment:
@mjrodgers I think you added this -- in how far is this a "fix"? Is it adding a missing API that is needed for generic programming or so? Just wondering what the use case is.

Reply:
This is quite old now, and I'm guessing it can be removed with the migration to the unified finite field type. I'm playing around with some things and seeing how to bring this experimental package up to date with some of the intervening OSCAR changes.